sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -8749,6 +9097,9 @@ public void testCommaJoinLeftFunction() throws Exception
     );
   }
 
+  // This SQL currently does not result in an optimum plan.
+  // Unfortunetly, we have disabled pushing down predicates (conditions and filters) due to https://github.com/apache/druid/pull/9773
+  // Hence, comma join will result in a cross join with filter on outermost
```

Comment:
Please update the docs to point out that people should avoid comma joins for this reason.

Reply:
Done